mover-syncthing/Dockerfile

```
@@ -0,0 +1,85 @@
+# Build the Syncthing image from Golang
+FROM golang:1.17 AS syncthing-builder
```

```suggestion
FROM registry.access.redhat.com/ubi8/go-toolset as builder
```
... to be consistent w/ the other movers